scripts/src/utils/utils.ts

```diff
@@ -187,3 +187,21 @@ export function dateParser(key: string, value: string) {
 
 	return value;
 }
+
+export async function batch(list: any[], chunkSize: number, delay: number, chunkCb: (chunk: any[], firstIndexOfChunk: number) => Promise<void>) {
```

Comment:
This whole function isn't inline with the rest of the coding standards in marketplace - i.e. using async/await instead of then for promises.
Also you have `console.log` and `setTimeout` which is very weird

Reply:
see https://github.com/kinecosystem/marketplace-server/pull/345/files#r292905889